File: Week_6/scores.py
# scores = [72, 73, 33]

# average = sum(scores)/len(scores)

# print(f"Average scores is {average}")

from cs50 import get_int

scores = []

for i in range(3):
    score = get_int("Score: ")
    scores.append(score)

average = sum(scores)/len(scores)

print(f"Average scores is {average}")

File: Week_2/scores.c
#include <cs50.h>
#include <stdio.h>

const int N = 3;
int main (void)
{
    int scores[N];
    for (int i = 0; i < N; i++)
    {
        scores[i] = get_int ("Scores: ");
    }
    printf ("Average: %f\n", (scores[0] + scores[1] + scores[2]) / (float) N);
}
